guides/source/active_record_callbacks.md

```diff
@@ -134,16 +173,199 @@ Here is a list with all the available Active Record callbacks, listed in the sam
+```
+
+#### Create Callbacks
+
+Create callbacks are triggered whenever the record is persisted (i.e. "saved")
+to the underlying database **for the first time**, in other words, when we're
+saving a new record, via the `create` or `save` methods. They are called before,
+after and around the object is created.
+
+```ruby
+class User < ApplicationRecord
+  before_create :set_default_role
+  around_create :log_creation
+  after_create :send_welcome_email
+
+  private
+    def set_default_role
+      self.role = "user"
+      Rails.logger.info("User role set to default: user")
+    end
+
+    def log_creation
+      Rails.logger.info("Creating user with email: #{email}")
+      yield
+      Rails.logger.info("User created with email: #{email}")
+    end
+
+    def send_welcome_email
+      UserMailer.welcome_email(self).deliver_later
+      Rails.logger.info("User welcome email sent to: #{email}")
+    end
+end
+```
+
+```irb
+irb> user = User.create(name: "John Doe", email: "john.doe@example.com")
+
+User role set to default: user
+Creating user with email: john.doe@example.com
+User created with email: john.doe@example.com
+User welcome email sent to: john.doe@example.com
+=> #<User id: 10, email: "john.doe@example.com", created_at: "2024-03-20 16:19:52.405195000 +0000", updated_at: "2024-03-20 16:19:52.405195000 +0000", name: "John Doe">
+```
+
+#### Using a Combination of Callbacks
+
+Often, you will need to use a combination of callbacks to achieve the desired
+behavior. For example, you may want to send a confirmation email after a user is
+created, but only if the user is new and not being updated. When a user is
+updated, you may want to notify an admin if critical information is changed. In
+this case, you can use `after_create` and `after_update` callbacks together.
+
+```ruby
+class User < ApplicationRecord
+  after_create :send_confirmation_email
+  after_update :notify_admin_if_critical_info_updated
+
+  private
+    def generate_confirmation_token
+      self.confirmation_token = SecureRandom.hex(10)
+      Rails.logger.info("Confirmation token generated for: #{email}")
+    end
+
+    def send_confirmation_email
+      UserMailer.confirmation_email(self).deliver_later
+      Rails.logger.info("Confirmation email sent to: #{email}")
+    end
+
+    def notify_admin_if_critical_info_updated
+      if saved_change_to_email? || saved_change_to_phone_number?
+        AdminMailer.user_critical_info_updated(self).deliver_later
+        Rails.logger.info("Notification sent to admin about critical info update for: #{email}")
+      end
+    end
+end
+```
+
+```irb
+irb> user = User.create(name: "John Doe", email: "john.doe@example.com")
+Confirmation email sent to: john.doe@example.com
+=> #<User id: 1, email: "john.doe@example.com", ...>
+
+irb> user.update(email: "john.doe.new@example.com")
+Notification sent to admin about critical info update for: john.doe@example.com
```

Comment:
I expect the updated value to be output to the log.
```suggestion
Notification sent to admin about critical info update for: john.doe.new@example.com
```

Reply:
NIce catch! Thank you!